src/analyze.ts

```diff
@@ -381,32 +392,31 @@ module.exports = async function (id, code, job) {
     let computed = computePureStaticValue(expression, true);
     if (!computed) return;
 
-    if (typeof computed.value === 'string') {
+    if ('value' in computed && typeof computed.value === 'string') {
       if (!computed.wildcards)
         (isImport ? imports : deps).add(computed.value);
       else if (computed.wildcards.length >= 1)
         emitWildcardRequire(computed.value);
     }
     else {
-      if (typeof computed.then === 'string')
+      if ('then' in computed && typeof computed.then === 'string')
         (isImport ? imports : deps).add(computed.then);
-      if (typeof computed.else === 'string')
+      if ('else' in computed && typeof computed.else === 'string')
         (isImport ? imports : deps).add(computed.else);
     }
   }
 
   let scope = attachScopes(ast, 'scope');
   handleWrappers(ast);
-  ({ ast = ast, scope = scope } = handleSpecialCases({ id, ast, scope, emitAsset: path => assets.add(path), emitAssetDirectory, job }) || {});
-
-  function backtrack (self, parent) {
+  handleSpecialCases({ id, ast, emitAsset: path => assets.add(path), emitAssetDirectory, job });
```

Comment:
Is it expected that handleSpecialCases() doesn't return anything? The previous code made it seem like it was supposed to return `ast` and `scope` but `handleSpecialCases` does not have a return statement.

Reply:
Yeah it's fine to remove the return paths.